src/Environment.ts

```diff
@@ -9,6 +9,7 @@ import { EnumOrString } from './EnumOrString';
  */
 export enum EnvironmentEnum {
   Production = 'PRODUCTION',
+  /** unsupported in the EU region */
```

Comment:
I forgot sbx was not supported in the EU. Should we update setupRequestUrl to throw an error if env = sbx and region = eu is passed in?

Reply:
Yeah, can edit it to throw an error

Reply:
Actually, I'll put it up as a different PR just so the it's reviewed separately since we'll probably need to edit the test files.